konfera/tests/test_models.py

```diff
@@ -178,6 +178,27 @@ def test_string_representation(self):
 
 
 class TicketTest(TestCase):
+    def setUp(self):
+        time = timezone.now()
+        location = models.Location(title='test_title', street='test_street', city='test_city', postcode=000000,
+                                   state='test_state', capacity=20)
+        location.save()
+        event = models.Event(title='test_event', description='test', event_type='meetup',
+                             status=models.event.PUBLISHED, location=location, date_from=time, date_to=time)
+        event.save()
+        self.ticket_type = models.TicketType(title='test', description='test', price=100, event=event,
+                                             date_from=timezone.now(), date_to=timezone.now())
```

Comment:
Not sure, I just copied and pasted from a different test that was constructing a new object :P I think we could use the variable. I'll change it here.